src/pcre2_stubs.c

```diff
@@ -293,7 +293,7 @@ CAMLprim value jit_match_unboxed(value ocaml_re /* : jit regex */, value subject
         // NOTE: need upper bound check here since we use pcre2_jit_match later and we want to
         // ensure we consistently return BADOFFSET rather than just failing to
         // find for JIT only.
```

Comment:
There are details of the PR comment that may be interesting to have in the code itself.

Reply:
I considered it, but I don't think it is that useful here.